src/mesh/RTT_Draco_Mesh_Reader.hh

```diff
@@ -0,0 +1,73 @@
+//----------------------------------*-C++-*----------------------------------//
+/*!
+ * \file   mesh/RTT_Draco_Mesh_Reader.hh
+ * \author Ryan Wollaeger <wollaeger@lanl.gov>
+ * \date   Friday, Jul 13, 2018, 08:38 am
+ * \brief  RTT_Draco_Mesh_Reader header file.
+ * \note   Copyright (C) 2018 Los Alamos National Security, LLC.
+ *         All rights reserved. */
+//---------------------------------------------------------------------------//
+
+#ifndef rtt_mesh_RTT_Draco_Mesh_Reader_hh
+#define rtt_mesh_RTT_Draco_Mesh_Reader_hh
+
+#include "Draco_Mesh_Reader.hh"
+#include "RTT_Format_Reader/RTT_Format_Reader.hh"
+
+namespace rtt_mesh {
+
+//===========================================================================//
+/*!
+ * \class RTT_Draco_Mesh_Reader
+ *
+ * \brief Wrap RTT file format reader and provide data to Draco_Mesh_Builder.
+ */
+//===========================================================================//
+
+class RTT_Draco_Mesh_Reader : public Draco_Mesh_Reader {
+private:
+  // >>> DATA
+
+  std::string filename;
```

Comment:
This can be const.

Reply:
Thank you for the recommendation.  This has been made const.